app/src/main/res/values/strings.xml

```diff
@@ -80,6 +80,7 @@
 
     <string name="post_username_format">\@%s</string>
     <string name="post_boosted_format">%s boosted</string>
+    <string name="post_replied_format">In reply to %s</string>
```

Comment:
```suggestion
    <string name="post_replied_format">%s replied</string>
```

?

Or is the %s here the name of the original author, and not the person who replied?

Reply:
The person who replied is the status owner (and thus listed one line below).

%s is the "replied to account name".